Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	RUSTFLAGS="-C debug-assertions" cargo test --release --features internals
```

Comment:
optional nit: line break @ EOF

Reply:
Added new line and updated the command as you suggested.  Thank you!